package.json

```diff
@@ -83,7 +78,8 @@
     "webpack": "2.6.1"
   },
   "scripts": {
-    "js": "babel source -d lib -s --no-babelrc --presets=env,react --plugins=transform-flow-strip-types,transform-decorators-legacy,transform-class-properties,lodash",
+    "js":
+      "babel source -d lib -s --no-babelrc --presets=env,react --plugins=transform-object-rest-spread,transform-flow-strip-types,transform-decorators-legacy,transform-class-properties,lodash",
```

Comment:
Agreed! I was running a linter that automatically formats as such, but that is obviously not a good place to let it loose. Is there is a certain linter/formatting configuration that the team is using which I could add to my text editor so I don't start reformatting 100's of lines of code with reckless abandon by accident? Commits that contain tons of formatting changes are basically illegible so I want to avoid that!